### ISSUE TITLE
Is realtime (Mercure, WebSocket...) supported?

### ISSUE THREAD

**Issue by dunglas** [Kong/swrv#16]
Hi, first of all thanks for this awesome library.

Is it possible to automatically trigger a refresh of the view when the data is mutated? For instance I use [Mercure](https://mercure.rocks) (an alternative to Websocket) to push new versions of the resources to the client. I would like the view to update when a new event is received, it works with SWR but it seems that it doesn't work (yet?) with SWRv. Here is what I tried:

```js
import { mutate } from 'swrv'

// ...
eventSource.onmessage = (e: MessageEvent) => {
      mutate(absoluteURL, Promise.resolve(JSON.parse(e.data)))
}
```

Mutate is called successfully, but unlike with SWR the currently displayed data doesn't update automatically. Am I missing something or is this not supported yet?

Best regards,

**Comment by darrenjennings** [comment]
@dunglas right now `mutate` in swrv is not actually triggering revalidations for all instances, it is only warming the cache for when a `swrv` instance goes to revalidate with the associate cache key. Sounds like we should aim to do this for feature parity!

**Comment by darrenjennings** [comment]
@dunglas merged in a #17 which should get you what you want. Let me know if you run into any issues.

demo:
https://codesandbox.io/s/swrv-16-20yc4
